doc/src/getting_started/install.rst

```diff
@@ -1,5 +1,17 @@
 .. _installation:
 
+SymPy Overview
+----------------
+
+SymPy is a computer algebra system (CAS) written in the Python programming language. SymPy is easy to use and install (see the download instructions and tutorial for more information), and works everywhere where Python 2.7 or newer is installed (Linux, Windows, Mac OS X, ...). SymPy's features include:
```

Comment:
This test is outdated. It would be better to take the first two paragraphs (about and sympy is...) from here:
https://www.sympy.org/en/index.html

Also it would be better at the top of the main index page rather than the install page.

Reply:
Moved all docs to index, trying to avoid a ToC for a section that is still this small, or you wanted the installation ToC @oscarbenjamin ?

Reply:
I don't understand what you are asking

Reply:
I don't understand what you are asking.

Reply:
Creating a separate index page then adding the installation page will lead to a small doc for the category plus a Table of contents for the installation section when you navigate to the getting-started section. Was this intended?

Reply:
re: Also it would be better at the top of the main index page rather than the install page.

Reply:
I meant to move this to the actual main index page. The very first page of the whole docs website (`doc/src/index.rst`).